### PR TITLE
DI-27154 configuration scheme for events

### PR DESCRIPTION
**Pull request by OdedAnswer**

### CODE REVIEW

File: addon.config.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "AddonUUID": "cbbc42ca-0f20-4ac8-b4c6-8f87ba7c16ad",
-  "AddonVersion": "0.7.5",
+  "AddonVersion": "0.7.6",
```
Comment (roiaharon-pepperi) [inline addon.config.json:3]: this needs to be changed to 0.8.0

File: shared/src/entities.ts [hunk 2]
```diff
@@ -31,6 +31,44 @@ export interface SelectOption<T> {
...
+        EventKey: {
+            Type: "String",
+        },
+        EventField: {
+            Type: "String",
+        },
+        EventFilter: {
+            Type: "Object",
+        },
+        LogicBlocks: {
```
Comment (roiaharon-pepperi) [inline shared/src/entities.ts:59]: please remove the LogicBlocks, this shouldn't be here

**Review by roiaharon-pepperi — CHANGES_REQUESTED** [review]

File: addon.config.json (hunk 1 above)
Comment (OdedAnswer) [reply addon.config.json:3]: right forgot

File: shared/src/entities.ts (hunk 2 above)
Comment (OdedAnswer) [reply shared/src/entities.ts:59]: but it is possible to have LogicBlocks, we said we allow them both and that they are mutually exclusive

File: addon.config.json (hunk 1 above)
Comment (roiaharon-pepperi) [reply addon.config.json:3]: no, you shouldn't publish new versions with old version number

**Review by roiaharon-pepperi — APPROVED** [review]

**Pull request merged**